src/open_inwoner/openzaak/migrations/0050_migrate_zgw_root_fields_to_multi_backend.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from django.db import migrations
+
+
+logger = logging.getLogger(__name__)
+
+
+def migrate_zgw_service_config_to_default_group_config(apps, schema_editor):
+    ZGWApiGroupConfig = apps.get_model("openzaak", "ZGWApiGroupConfig")
+    OpenZaakConfig = apps.get_model("openzaak", "OpenZaakConfig")
+
+    for config in OpenZaakConfig.objects.all():
```

Comment:
I don't think looping here is necessary, because `OpenZaakConfig` is a `SingletonModel`, but it's not a big issue